src/main/java/Todo.java

```diff
@@ -0,0 +1,34 @@
+public class Todo extends Task {
+    public Todo(String description) {
+        super(description);
+        type = TaskType.TODO;
+    }
+
+    public static Todo fromTaskString(String taskString) {
+        String[] parts = taskString.split("\\|");
+        if (parts.length < 3)
+            return null;
```

Comment:
Perhaps you should add open and close braces Egyptian style to comply with the coding standard.

Reply:
Noted, thanks for pointing this out!